test/TorchSharpTest/NN/NNModuleTest.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using TorchSharp.JIT;
+using TorchSharp.NN;
+using TorchSharp.Tensor;
+using Xunit;
+
+namespace TorchSharpTest.NN
+{
+    public class NNModuleTest
```

Comment:
Can you please move this tests into TorchSharp test? Or alternatively, can you please move all NN modules tests in TorchSharp into NNModuleTest?

Reply:
done moved test cases to TorchSharp test